rs_inverse/scripts/stand_mode_spot.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
Add comment to explain what this file is doing

Reply:
added in 27457314be434390090a38690c5ee955afa4c1c6